main.py

```diff
@@ -1,27 +1,36 @@
-import PySimpleGUI as sg      
+import PySimpleGUI as sg    
+from json import dumps 
 
 sg.theme("LightGray1")
-print("Distância")
```

Comment:
Você usa input qdo for fazer um programa em linha de comando.
No caso do PySimpleGUI você pode fazer de 'janelas'.